backend/Cargo.toml

```diff
@@ -59,3 +59,6 @@ image = { version = "0.24.6", default-features = false, features = ["png"] }
 
 [dev-dependencies]
 jsonwebkey = { version = "0.3.5", features = ["generate", "jwt-convert"] }
+
+[profile.release]
+debug = true
```

Comment:
I described it in doc. And yes this will be removed again (part of the reason why PR is draft), its just easier without having to exclude it from git every time.

Reply:
Then add a TODO, then it is also clear for reviewers.